crates/revm/src/processor.rs

```diff
@@ -57,9 +56,7 @@ pub struct EVMProcessor<'a, EvmConfig> {
     /// The configured chain-spec
     pub(crate) chain_spec: Arc<ChainSpec>,
     /// revm instance that contains database and env environment.
-    pub(crate) evm: EVM<StateDBBox<'a, ProviderError>>,
-    /// Hook and inspector stack that we want to invoke on that hook.
-    stack: InspectorStack,
+    pub(crate) evm: Option<Evm<'a, InspectorStack, StateDBBox<'a, ProviderError>>>,
```

Comment:
Why is the EVM optional now?

Reply:
It was made so we can mutate `Evm` in `EvmBuilder` as I needed to change a generic of `external Context` to `InspectorStack`. But on the second iteration, I just embedded `InspectorStack` inside Evm always.

I should just add a few functions to `Handler` and there will be no need to mutate `Evm` and no need for `Option` 

cc @Rjected

Reply:
done, the `Option` is removed. Added few functions directly to `revm::Handler`